build.gradle

```diff
@@ -473,6 +476,15 @@ task mapNamedJar(dependsOn: [buildTinyWithEnum, mapIntermediaryJar]) {
 	}
 }
 
+task decompileCFR(dependsOn: mapNamedJar, type: JavaExec) {
+	doFirst {
+		file('src').deleteDir()
+	}
+
+	classpath = configurations.cfrRuntime
+	main = 'org.benf.cfr.reader.Main'
+	args '--outputdir', 'src', namedJar.getAbsolutePath()
```

Comment:
Given we have already get the libraries when we've map the jar, we can just supply the libraries to the decompiler. The flag is `--extraclasspath`, and it seems to take a parameter like java's classpath (a string)

Reply:
It seems to decompile just fine without it, ill look into setting that up but I dont think its needed.

Reply:
Done.